doc/source/tutorials.rst

```diff
@@ -0,0 +1,256 @@
+2. The optimal angles for the single QAOA layer are found and will be used.
+
+With these assumptions met, we provide an example circuit `c` below is written in IQM Garnet's native gates with specific qubits. The parameters for the `PRX` gates are optimal. We select `verbatim_circuit=True` as we do not want the device to transpile the circuit.
+
+.. code-block:: python
+
+   c = QiboCircuit(10):
+   c.add(gates.PRX(3, -np.pi, np.pi/2))
+   c.add(gates.PRX(3, np.pi, -np.pi/2))
+   c.add(gates.PRX(4, np.pi/2, np.pi/2))
+   c.add(gates.PRX(4, np.pi, 0))
+   c.add(gates.CZ(3, 4))
+   c.add(gates.PRX(3, -1.081592653589793, 0))
+   c.add(gates.PRX(3, np.pi, -np.pi))
+   c.add(gates.CZ(3, 4))
+   c.add(gates.PRX(3, np.pi/2, np.pi/2))
+   c.add(gates.PRX(3, np.pi, 0))
+   c.add(gates.PRX(5, -np.pi, np.pi/2))
+   c.add(gates.PRX(5, np.pi, -np.pi/2))
+   c.add(gates.PRX(9, -np.pi, np.pi/2))
+   c.add(gates.PRX(9, np.pi, -np.pi/2))
+   c.add(gates.CZ(4, 9))
+   c.add(gates.PRX(9, -1.081592653589793, 0))
+   c.add(gates.PRX(9, np.pi, -np.pi))
+   c.add(gates.CZ(4, 9))
+   c.add(gates.CZ(4, 5))
+   c.add(gates.PRX(5, -1.081592653589793, 0))
+   c.add(gates.PRX(5, np.pi, -np.pi))
+   c.add(gates.CZ(4, 5))
+   c.add(gates.PRX(4, 2.850796326794897, 0))
+   c.add(gates.PRX(5, -np.pi, np.pi/2))
+   c.add(gates.PRX(5, np.pi, -np.pi/2))
+   c.add(gates.PRX(9, -np.pi/2, -np.pi))
+   c.add(gates.PRX(9, np.pi, -np.pi/4))
+   c.add(gates.CZ(4, 9))
+   c.add(gates.PRX(4, np.pi/2, 0))
+   c.add(gates.PRX(9, np.pi/2, 0))
+   c.add(gates.CZ(4, 9))
+   c.add(gates.PRX(4, np.pi/2, 0))
+   c.add(gates.PRX(9, np.pi/2, 0))
+   c.add(gates.CZ(4, 9))
+   c.add(gates.PRX(4, np.pi/2, np.pi/2))
+   c.add(gates.PRX(4, np.pi, 0))
+   c.add(gates.CZ(3, 4))
+   c.add(gates.PRX(4, -1.081592653589793, 0))
+   c.add(gates.PRX(4, np.pi, -np.pi))
+   c.add(gates.CZ(3, 4))
+   c.add(gates.PRX(3, 1.28, 0))
+   c.add(gates.PRX(4, np.pi/2, np.pi/2))
+   c.add(gates.PRX(4, np.pi, 0))
+   c.add(gates.CZ(4, 5))
+   c.add(gates.PRX(4, -1.081592653589793, 0))
+   c.add(gates.PRX(4, np.pi, -np.pi))
+   c.add(gates.CZ(4, 5))
+   c.add(gates.PRX(4, 1.28, 0))
+   c.add(gates.PRX(5, -np.pi/2, -2.850796326794897))
+   c.add(gates.PRX(5, np.pi, -0.64)
+   c.add(gates.M(9))
+   c.add(gates.M(3))
+   c.add(gates.M(4))
+   c.add(gates.M(5))
+
+We define the problem Hamiltonian of QAOA for MaxCut, `obs`. The `obs` has to be written according to the qubit mapping applied for circuit `c`.
+
+.. code-block:: python
+
+   obs = 2.5 - 0.5*Z(3)*Z(9) - 0.5*Z(4)*Z(3) - 0.5*Z(4)*Z(5) - 0.5*Z(4)*Z(9) - 0.5*Z(9)*Z(5)
+   obs = SymbolicHamiltonian(obs, nqubits=c.nqubits, backend=NumpyBackend())
+
+Finally, we can run ZNE by setting the backend to the `AWS` to obtain the estimated (extrapolated) result.
+
+.. code-block:: python
+
+   device = AwsDevice('arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet')
+   AWS = BraketClientBackend(device = device, verbatim_circuit=True)
```

Comment:
```suggestion
```
Again, I would avoid repeating this over and over. I would just define it on top and, as long as it is unchanged, refer to it in the following.